ts/src/pro/coinbaseinternational.ts

```diff
@@ -481,7 +483,6 @@ export default class coinbaseinternational extends coinbaseinternationalRest {
          * @param {object} [params] extra parameters specific to the exchange API endpoint
          * @returns {object} A dictionary of [order book structures]{@link https://docs.ccxt.com/#/?id=order-book-structure} indexed by market symbols
          */
```

Comment:
rule of thumb we should loadMarkets at the beginning of the unified methods, in the future we might use something other than subscribeMultiple